extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -282,23 +357,33 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
    * @private
    */
   openRemote_() {
-    const animationEl = this.win.document.createElement('div');
-    animationEl.classList.add('i-amphtml-story-page-attachment-expand');
-    const storyEl = closest(this.element, (el) => el.tagName === 'AMP-STORY');
+    const clickTarget = this.element.parentElement
+      .querySelector('.i-amphtml-story-page-open-attachment-host')
+      .shadowRoot.querySelector('a.i-amphtml-story-page-open-attachment');
+
+    const isMobileUI =
+      this.storeService_.get(StateProperty.UI_STATE) === UIType.MOBILE;
+    // Shows outlink url preview on mobile only.
+    if (!isMobileUI) {
+      triggerClickFromLightDom(clickTarget, this.element);
+    } else {
+      const animationEl = this.win.document.createElement('div');
+      animationEl.classList.add('i-amphtml-story-page-attachment-expand');
```

Comment:
is there a reason why this is not inside the `mutate` phase below? I'm guessing it messes up the animation but just wanted to make sure.